src/Link.js

```diff
@@ -17,31 +16,26 @@ const buttonStyles = {
   appearance: 'none'
 }
 
-const hoverColor = system({
-  hoverColor: {
-    property: 'color',
-    scale: 'colors'
-  }
-})
-
-const Link = styled.a`
+const Link = styled.a.attrs(props => ({
+  color: props.color ? props.color : props.muted ? 'gray.6' : 'blue.5'
```

Comment:
I don't believe this will work - if someone passes in `blue.3` to `props.color` this won't grab the color from the theme - we'll need to do something like:
```suggestion
  color: props.color ? `${get(props.color)}` : props.muted ? 'gray.6' : 'blue.5'
```

I'm _pretty_ sure that would work but make sure to double check!

Reply:
I also don't think this needs to be in `attrs`

Reply:
Ooooh nevermind - I see what you're doing :) This is fine

Reply:
I tested this out locally and it looks like this isn't actually getting picked up from styled system, I believe it needs to be moved out of attrs and handled like the other styles